lib/menu.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Menu do
+  def game_types do
+    display_game_types
+
+    get_value_betweenf(0, Enum.count(games) - 1)
+    |> get_players
+  end
+
+  defp display_game_types do
+    IO.puts Enum.map(games(), fn({game_type, _}) -> game_type <> "\n" end)
```

Comment:
This could be:
games()
|> Enum.map(fn({type, _}) -> type end)
|> Enum.join("\n")
|> IO.put